java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java

```diff
@@ -306,7 +306,13 @@
 
   // Pattern to capture digits used in an extension. Places a maximum length of "7" for an
   // extension.
-  private static final String CAPTURING_EXTN_DIGITS = "(" + DIGITS + "{1,7})";
+  private static final String CAPTURING_EXTN_DIGITS = "(\\p{Nd}{1,7})";
+
+  // Pattern to capture digits and commands used in an extension. Places a maximum length of "7"
+  // for an entire extension. Comma and semicolon are allowed, as long as they are not the trailing symbol.
+  // They denote dialing commands to delay (comma) or wait for a tone (semicolon).
```

Comment:
Please reference the standards (URL?) which mention comma and semicolon for this purpose.

Reply:
This is part of ITU-T V.250 https://www.itu.int/rec/T-REC-V.250-200307-I/en

Specifically 6.3.1.2 Pause during dialling, and 5.4.5.1 Concatenating commands after extended syntax commands, from https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-V.250-200307-I!!PDF-E&type=items